velox/external/date/date.h

```diff
@@ -396,7 +396,7 @@ operator<<(std::basic_ostream<CharT, Traits>& os, const month& m);
 
 class year
 {
-    short y_;
+    int y_;
```

Comment:
Where date.h is coming from? Would you open an issue in the respective GitHub repo and add a link here?

Reply:
Is it coming from https://github.com/HowardHinnant/date/issues ?

Reply:
Yes, it's copy of https://github.com/HowardHinnant/date/ to velox repo a while ago, to be used in the same way as Presto Java uses joda datatime lib.

Will file an Issue to the GitHub repo and link here.